pkg/config/kind.go

```diff
@@ -35,7 +35,7 @@ const (
 	ListsKind
 	MetricsKind
 	QuotasKind
-
+	Undefined
```

Comment:
Why not make this the first in the list (following proto enum convention of `*_UNSPECIFIED = 0`, etc.)